tests/functional/object/rgw/test_rgw_sts.py

```
@@ -0,0 +1,534 @@
+        )
+
+        logger.info("CephObjectStore STS configuration verified successfully")
+
+        # Step 4: Create IAM user and bucket
+        logger.info("Step 4: Creating IAM user and S3 bucket")
+        iam_user_name = create_unique_resource_name("sts-iam", "user")
+        bucket_name = create_unique_resource_name("sts-test", "bucket")
+
+        try:
+            # Create IAM user
+            logger.info(f"Creating IAM user: {iam_user_name}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)